src/coreclr/src/jit/compiler.h

```diff
@@ -9161,6 +9161,36 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
         return varTypeIsStruct(info.compRetNativeType) && (info.compRetBuffArg == BAD_VAR_NUM);
 #endif // TARGET_XXX
 
+#else // not FEATURE_MULTIREG_RET
+
+        // For this architecture there are no multireg returns
+        return false;
+
+#endif // FEATURE_MULTIREG_RET
+    }
+
+    // Returns true if the method returns a value in more than one return register,
+    // it should replace/be  merged with compMethodReturnsMultiRegRetType when #36868 is fixed.
+    bool compMethodReturnsResInMultiplyRegisters()
+    {
+#if FEATURE_MULTIREG_RET
+#if defined(TARGET_X86)
+        // On x86 only 64-bit longs are returned in multiple registers
+        return varTypeIsLong(info.compRetNativeType);
+#else // targets: X64-UNIX, ARM64 or ARM32
```

Comment:
This could be an `#elif`?
Also, isn't it the case that ARM32 also returns 64-bit longs in multiple registers?

Reply:
You are right, it is also missed here (and in the original function), added note to #36868.

Reply:
> This could be an #elif?

I need a separate block for `TARGET_ARM64` and then a common for `arm64, x64 Unix, arm`, so I do not see how to use `elif` here.